src/components/Profile/AboutMe/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Check, PencilAlt } from '@styled-icons/fa-solid'
+import React, { useState } from 'react'
+import { Description, Edit, EditDescription } from '../styled'
+
+export const AboutMe: React.FC = () => {
+	const [element, setElement] = useState<boolean>(true)
+	const [text, setText] = useState<string>('')
+
+	return (
+		<>
+			<h2>
+				Něco o mně
+				{!element && (
+					<Edit onClick={() => setElement(true)}>
+						<PencilAlt />
+					</Edit>
+				)}
+			</h2>
+			{element ? (
+				<>
+					<EditDescription
+						value={text}
+						placeholder="Zde můžeš napsat něco o sobě . . ."
+						onChange={(e) => setText(e.target.value)}
+					/>
+					<Edit
```

Comment:
I would float the edit to the right (`float: right;`). Editing name is on the right, adding new social network also has button on the right, so this seems a bit out of place.

Reply:
The Edit component is used in many places, and if I add a float rigth, I break the layout of the others.